.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug, needs triage
+assignees: ''
+
+---
+
```

Comment:
Shall we add a link to our contributing guidelines at the beginning?

Reply:
Not sure, I would say it's not necessary to read the contribution guidelines to file an issue. I see that more related to a real contribution.